src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/AdjudicationsResourceTest.java

```diff
@@ -399,4 +400,149 @@ public void returns403IfInvalidRole() {
             assertThatStatus(response, 403);
         }
     }
+
+    @Nested
+    public class AdjudicationHearings {
+
+        final String token = validToken(List.of("ROLE_MAINTAIN_ADJUDICATIONS"));
+        final String invalidToken = validToken(List.of("ROLE_SYSTEM_USER"));
+        final HttpEntity<?> validRequest = createHttpEntity(token, Map.of("dateTimeOfHearing","2022-10-24T10:12:44", "hearingLocationId", "-31", "oicHearingType", "GOV_ADULT"));
+        final HttpEntity<?> invalidRequest = createHttpEntity(token, Map.of("dateTimeOfHearing","not a date time", "hearingLocationId", "-31", "oicHearingType", "GOV_ADULT"));
+        final HttpEntity<?> invalidLocationRequest = createHttpEntity(token, Map.of("dateTimeOfHearing","2022-10-24T10:12:44", "hearingLocationId", "1", "oicHearingType", "GOV_ADULT"));
+        final HttpEntity<?> invalidRoleRequest = createHttpEntity(invalidToken, Map.of("dateTimeOfHearing","2022-10-24T10:12:44", "hearingLocationId", "1", "oicHearingType", "GOV_ADULT"));
+        final HttpEntity<?> invalidTypeRequest = createHttpEntity(token, Map.of("dateTimeOfHearing","2022-10-24T10:12:44", "hearingLocationId", "-31", "oicHearingType", "WRONG"));
+
+
+        @Test
+        public void createHearingReturns403ForInvalidRoles () {
+            final var response = testRestTemplate.exchange(
+                "/api/adjudications/adjudication/-9/hearing",
+                HttpMethod.POST,
+                invalidRoleRequest,
+                new ParameterizedTypeReference<String>() {
+                });
+
+            assertThatStatus(response, 403);
+        }
+        @Test
+        public void createHearingReturns404DueToNoAdjudication() {
+            final var response = testRestTemplate.exchange(
+                "/api/adjudications/adjudication/99/hearing",
+                HttpMethod.POST,
+                validRequest,
+                new ParameterizedTypeReference<String>() {
+                });
+
+            assertThatStatus(response, 404);
+        }
+
+        @Test
+        public void createHearingReturns400() {
+            final var response = testRestTemplate.exchange(
+                "/api/adjudications/adjudication/-9/hearing",
+                HttpMethod.POST,
+                invalidRequest,
+                new ParameterizedTypeReference<String>() {
+                });
+
+            assertThatStatus(response, 400);
+        }
+
+        @Test
+        public void createHearingReturns400forOicHearingType() {
```

Comment:
very minor style observation: I wonder if a test name of something like `willReturn400WhenHearingTypeIsNotValid` or something similar would be more descriptive